cmd/build/helmify/static/values.yaml

```diff
@@ -65,6 +65,11 @@ controllerManager:
     requests:
       cpu: 100m
       memory: 256Mi
+  autoscaling:
```

Comment:
please  update the helmify readme with these

Reply:
updated